ui/src/app/pages/admin/workspace/basic-information.tsx

```diff
@@ -35,7 +35,10 @@ export const BasicInformation = ({
             {workspace.initialCredits?.exhausted
               ? 'Exhausted'
               : 'Not Exhausted'}
-            , {workspace.initialCredits?.expired ? 'Expired' : 'Not Expired'}
```

Comment:
It seems like this is the only place where the attribute was still used in the UI. The rest are in tests. What do you think about doing api and UI un the same commit?